agent/agent-wrapper.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+$listenOn = getenv('NIGHTWATCH_INGEST_URI') ?: '0.0.0.0:2407';
+$refreshToken = getenv('NIGHTWATCH_TOKEN');
```

Comment:
I'm reasonably sure this should work already. In the agent we do `$refreshToken ??= $_SERVER['NIGHTWATCH_TOKEN'] ?? '';`

If not, lets make it work.